crates/wasmparser/compare-with-main.sh

```diff
@@ -1,10 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Happy that this file is gone. I lost work multiple times since apparently it clears the git workspace.